src/components/Header/Header.test.tsx

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
 import { Header } from './Header';
+import { BrowserRouter } from 'react-router-dom';
 
 describe('The Header component', () => {
   it('renders successfully', () => {
-    render(<Header />);
+    render(
```

Comment:
what is the intention of this test?

Reply:
Just to make sure that it renders without crashing. I suppose it would make sense to add a check that it forms the URLs to pages correctly with the new feed prop passed to it?

Reply:
Let me rebase and resolve conflicts as well... Right now the custom Button doesn't want to play nice with the `component` prop but should be passing this through without any issues.

Reply:
@kkyeboah, I've rebased the branch and resolved the conflicts + added a test checking how Header component renders URLs. Please review when you have time.